### DISCUSSION TITLE
Proxy Addresses and Amazon

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#74] [Q&A]
I got about 5 rotation IP addresses with Webshare.io and they were already pre-banned with Amazon. Webshare rotates their IPs but I have a proxy.txt with a list of IPs so if it rotates, I am not going to know. Is there a way it can rotate in StockAlert? (i.e. p.webshare.io?)...

**Comment by Prince25** [comment]
Sorry, it's not totally clear what you're trying to ask. Are you asking if there's a way for StockAlertBot to automatically use Webshare's Automatic Refreshes feature in which they automatically refresh your entire proxy list? Or are you talking about their rotating proxies? If it's the latter, then I believe you can use the same IP, `p.webshare.io`, and Webshare automatically selects a random from your proxy list. You can change the `Rotation Frequency` so it refreshes the proxies it uses.
